scripts/gui.lua

```diff
@@ -590,7 +597,55 @@ function GUI.onTextChanged(event)
             GUI.updateTrackerConfig(playerSettings.gui.trackerInfo, selectedTracker)
             GUI.updateTrackerInfo(playerSettings.gui.trackerInfo, selectedTracker)
         end
-    end
+    elseif event.element.name == "tlbe-tracker-cityblock-size-x" then
+        local selectedTracker = playerSettings.trackers[playerSettings.guiPersist.selectedTracker]
```

Comment:
Can we move this outside of the if-elseif-end block? (It is duplicated a lot...)

IIRC `player` and playerSettings` are on top of this function, it would be a nice place for `selectedTracker` as well.

Reply:
I agree there's a lot of redundancy here. Is selectedTracker well-defined for all events though? I wasn't sure if it would fail or contain junk if we move it out to the top level.

Reply:
I think there will/should be always a tracker (and a camera) selected, or the GUI would break as there is nothing to show on the right side(s) of the GUI. Additionally, if it would contain junk: it should be for an event/trigger and is not using it... (hopefully?)

Reply:
Done